contracts/test/abi/AggregatorInterfaceAbi.json

```diff
@@ -0,0 +1,113 @@
+[
```

Comment:
I don't think this is needed here, since we already have the interface file under `/contracts/interfaces` directory. You can just do `ethers.getContractAt("AggregatorV3Interface", address)` and it'll work

Reply:
that is a great point thanks! removed